raiden/tests/integration/contracts/test_token_network.py

```diff
@@ -19,12 +20,68 @@
     DuplicatedChannelError,
     TransactionThrew,
     ChannelIncorrectStateError,
+    DepositMismatch,
 )
 from raiden.tests.utils import wait_blocks
 from raiden_libs.messages import BalanceProof
 from raiden_libs.utils.signing import sign_data
 
 
+def test_token_network_deposit_race(
+    token_network_proxy,
+    private_keys,
+    blockchain_rpc_ports,
+    token_proxy,
+    chain_id,
+    web3,
```

Comment:
8 spaces :)

Reply:
Ah ffs. I used to be good at the styling by eye stuff. What happened to me ... how did I not see it